tests/test_zinc_group.py

```diff
@@ -110,3 +110,15 @@ def test_group_add_compare_group_local_contents(self):
         self.assertEqual(0, group4.getMeshGroup(mesh1d).getSize())
         self.assertEqual(1, group4.getNodesetGroup(nodes).getSize())
         self.assertTrue(groups_have_same_local_contents(group3, group4))
+
+        group_remove_group_local_contents(group2, group3)
+        self.assertEqual(1, group2.getMeshGroup(mesh3d).getSize())
+        self.assertEqual(5, group2.getMeshGroup(mesh2d).getSize())
+        self.assertEqual(12, group2.getMeshGroup(mesh1d).getSize())
+        self.assertEqual(7, group2.getNodesetGroup(nodes).getSize())
+
+        group_remove_group_local_contents(group2, group1)
+        self.assertEqual(0, group2.getMeshGroup(mesh3d).getSize())
+        self.assertEqual(0, group2.getMeshGroup(mesh2d).getSize())
+        self.assertEqual(0, group2.getMeshGroup(mesh1d).getSize())
+        self.assertEqual(0, group2.getNodesetGroup(nodes).getSize())
```

Comment:
Do we also want to see here the current state of group1 for these same groups?

Reply:
Done